cmd/nerdctl/system_prune.go

```diff
@@ -99,5 +104,29 @@ func systemPruneAction(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
-	return imagePrune(cmd, client, ctx)
+	if err := imagePrune(cmd, client, ctx); err != nil {
+		return err
+	}
+
+	var buf bytes.Buffer
+	outWriter := bufio.NewWriter(&buf)
+	copiedCmd := *cmd
+	copiedCmd.SetOut(outWriter)
+	if err := builderPruneAction(&copiedCmd, args); err != nil {
```

Comment:
Why not make `builder prune` the same as docker?

```sh
$ sudo docker builder prune
WARNING! This will remove all dangling build cache. Are you sure you want to continue? [y/N] y
Deleted build cache objects:
mzl53zo5788bl0bzvz1j72j1g
yp82akq9eitd5jb5ni6k9pvd4
ik2txugxyg3ik669w77ho4pqq
sd78ank4ncfdhqjilztfbwqik
```

Reply:
I would make it. But I meet some trouble when parsing the output of `buildctl prune`. 🥲

Reply:
https://github.com/containerd/nerdctl/pull/1284#issuecomment-1203933222